contrib/containerlab/bgpv2/auto-discovery/default-gateway/multi-homing/Makefile

```diff
@@ -0,0 +1,25 @@
+VERSION ?= v1.16.0
+
+deploy:
+	kind create cluster --config cluster.yaml
+	sudo containerlab -t topo.yaml deploy
+	# create secret for bgp
+	kubectl -n kube-system create secret generic --type=string bgp-auth-secret --from-literal=password=cilium123
+	# install cilium
+	#helm install cilium -n kube-system cilium/cilium --version $(VERSION) -f values.yaml
+	#cilium status --wait --namespace kube-system
```

Comment:
Looks like cilium installation is skipped in this lab. How about installing it using the local chart and dev images instead of fixed version? (same for the other lab).

Reply:
thanks for pointing this out. i update the Makefile.